src/cobra/core/gene.py

```diff
@@ -154,7 +213,7 @@ def __init__(self, id=None, name="", functional=True):
         self._functional = functional
```

Comment:
Line above should be

```py
super().__init__(id=id, name=name, **kwargs)
```

Reply:
Species doesn't have **kwargs, so I made it slightly different